packages/stan/package.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name": "@wordpress/stan",
+	"version": "0.0.1",
+	"description": "WordPress state library.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"gutenberg",
+        "state",
```

Comment:
Some formatting issue, we should enable Prettier for `json` and `md` and fix it later 😄